coding-concepts.js

```diff
@@ -5,13 +5,22 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
+
+
+
+
 // --------------------1) What will this log?
 
 var cohort = "Alpha 2022"
 // console.log(cohort.split(""))
 
 // a) Your answer:
-// b) Verify and explain:
+//This will split the string by every individual character, including assigning the space between the two words as its own character.
+
+// b) Verify and explain: From what I've learned by playing around with the split method previously, it splits a string by removing what is stated between the quotation marks and grouping what is on either side. For example, a simple argument wouldbe "" because I see it as removing the tiny space between the letters and defining that as where to split everything, leading to each individual letter being a string. Similarly, putting a space between the quotes in the argument removes the space and would call "Alpha" and "2022" different strings. Even stranger would be if I put "a" in the argument because .split would remove "a" from the string and return "Alph" " 2022" because it was split at the occurrence at the lower case "a".
```

Comment:
Love that you play around with the techniques that will certainly help with how you approach coding